Marlin/Marlin_main.cpp

```diff
@@ -2702,30 +2708,80 @@ inline void gcode_G28() {
             SERIAL_EOL;
           } // yy
           SERIAL_EOL;
-          if (verbose_level > 3) {
-            SERIAL_PROTOCOLPGM(" \nCorrected Bed Height vs. Bed Topology: \n");
-
-            for (int yy = auto_bed_leveling_grid_points - 1; yy >= 0; yy--) {
-              for (int xx = 0; xx < auto_bed_leveling_grid_points; xx++) {
-                int ind = yy * auto_bed_leveling_grid_points + xx;
-                float x_tmp = eqnAMatrix[ind + 0 * abl2],
-                  y_tmp = eqnAMatrix[ind + 1 * abl2],
-                  z_tmp = 0;
-
-                apply_rotation_xyz(plan_bed_level_matrix,x_tmp,y_tmp,z_tmp);
-
-                float diff = eqnBVector[ind] - z_tmp - min_diff;
-                if (diff >= 0.0)
-                  SERIAL_PROTOCOLPGM(" +");
-                // Include + for column alignment
-                else
-                  SERIAL_PROTOCOLCHAR(' ');
-                SERIAL_PROTOCOL_F(diff, 5);
-              } // xx
-              SERIAL_EOL;
-            } // yy
+
+          SERIAL_PROTOCOLPGM(" \nCorrected Bed Topography: \n");
+          for (int yy = auto_bed_leveling_grid_points - 1; yy >= 0; yy--) {
+            for (int xx = 0; xx < auto_bed_leveling_grid_points; xx++) {
+              int ind = yy * auto_bed_leveling_grid_points + xx;
+              float diff = eqnBVector[ind] - min_diff;
+              if (diff >= 0.0)
+                SERIAL_PROTOCOLPGM(" +");   // Include + for column alignment
+              else
+                SERIAL_PROTOCOLCHAR(' ');
+              SERIAL_PROTOCOL_F(diff, 5);
+            } // xx
             SERIAL_EOL;
-          }
+          } // yy
+          SERIAL_EOL;
+
+          /*
+          SERIAL_PROTOCOLPGM(" \nBed Topography in new coordinats:\n");
+                for (int yy = auto_bed_leveling_grid_points - 1; yy >= 0; yy--) {
+                  for (int xx = 0; xx < auto_bed_leveling_grid_points; xx++) {
+                    int ind = yy * auto_bed_leveling_grid_points + xx;
+
+                    vector_3 probe_point = vector_3(eqnAMatrix[ind + 0 * abl2], eqnAMatrix[ind + 1 * abl2], eqnBVector[ind]);
+                    probe_point.apply_rotation(inverse_bed_level_matrix);
+                    float diff = probe_point.z;
+                    if (diff >= 0.0)
+                      SERIAL_PROTOCOLPGM(" +");
+                    // Include + for column alignment
+                    else
+                      SERIAL_PROTOCOLCHAR(' ');
+                    SERIAL_PROTOCOL_F(diff, 5);
+                  } // xx
+                  SERIAL_EOL;
+                } // yy
+                SERIAL_EOL;
+          */
+          SERIAL_PROTOCOLPGM(" \nCorrected Bed Topography in new coordinats:\n");
```

Comment:
correction: "coordinates"

Actually these strings can be made shorter and to use less PRAM. If the string "\nCorrected Bed Topography " is printed separately, it will take half as much space as including it in two strings. So, above you would have…
```cpp
SERIAL_PROTOCOLPGM(" \nCorrected Bed Topography");
SERIAL_PROTOCOLLNPGM(": ");
```
…and here you would have…
```cpp
SERIAL_PROTOCOLPGM(" \nCorrected Bed Topography");
SERIAL_PROTOCOLLNPGM(" in new coordinates:");
```